packages/nx/src/command-line/login/command-object.ts

```diff
@@ -0,0 +1,23 @@
+import { Argv, CommandModule } from 'yargs';
+
+export const yargsLoginCommand: CommandModule = {
+  command: 'login [nxCloudUrl]',
+  describe:
+    'Login to Nx Cloud by generating a personal access token and saving it to your local machine.',
+  builder: (yargs) =>
+    yargs
+      .positional('nxCloudUrl', {
+        describe:
+          'The Nx Cloud URL of the instance you are trying to connect to. If no positional argument is provided, this command will connect to https://cloud.nx.app.',
+        type: 'string',
+        required: false,
+      })
+      .option('verbose', {
```

Comment:
We have a helper function, `withVerbose` which will add the `verbose` option.

It includes the middleware to set the environment variable.

You can see it used here: https://github.com/nrwl/nx/blob/c72ba9b50417d7739a3edf9bdae21b74beccc7bd/packages/nx/src/command-line/import/command-object.ts#L41